tracing-core/src/lib.rs

```diff
@@ -91,7 +91,7 @@
     issue_tracker_base_url = "https://github.com/tokio-rs/tracing/issues/"
 )]
 #![cfg_attr(not(feature = "std"), no_std)]
-#![cfg_attr(docsrs, feature(doc_cfg))]
+#![cfg_attr(docsrs, feature(doc_cfg), deny(broken_intra_doc_links))]
```

Comment:
huh, i've never seen this form of `cfg_attr` with multiple attributes before, does this work?

Reply:
I didn't even know about it until a while ago, but https://github.com/rust-lang/rust/pull/54862 Implemented cfg_attr with multiple attributes!